scripts/globals/mobskills/Bubble_Shower.lua

```diff
@@ -10,6 +10,23 @@ require("scripts/globals/monstertpmoves");
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
+	mob = mob:getID();
+	if (mob == 17129519) then

```

Comment:
@bendangelo didn't mean to tag you, was about to ask a question when I realized the answer: you made a mobMod for it (which was awesome, thanks)